manager/assets/modext/widgets/system/modx.grid.context.js

```diff
@@ -8,6 +8,17 @@
  */
 MODx.grid.Context = function(config) {
     config = config || {};
+    this.buttonColumnTpl = new Ext.XTemplate('<tpl for=".">'
+        + '<tpl if="action_buttons !== null">'
+        + '<ul class="x-grid-buttons">'
+        + '<tpl for="action_buttons">'
+        + '<li><i class="icon {className} icon-{icon}" title="{text}"></i></li>'
```

Comment:
The variables should be escaped to prevent XSS, it's been a while but I think the syntax in extjs was `{className|htmlEncode}` or something similar?